app/controllers/case_management/tax_returns_controller.rb

```diff
@@ -12,7 +12,7 @@ class TaxReturnsController < ApplicationController
     def edit; end
 
     def update
-      @tax_return.update!(tax_return_params)
+      @tax_return.update!(assign_params)
```

Comment:
If the method was called `tax_return_params` or `update_params` then CanCanCan should automatically have assigned the new attributes (for example calling `@tax_return.assign_attributes(update_params)`) In which case yo'd only have to call `@tax_return.save!`

Reply:
1) i didn't know that and thats really cool. 
2) this controller has TWO update methods -- one that only updates the status and one that only updates the assigned user. I kind of thing it makes sense to have them in separate controllers, so I could potentially refactor into two separate controllers.